src/Polly.Net35/CircuitBreaker/SuccessRatioCircuitBreakerState.cs

```diff
@@ -55,10 +64,15 @@ public void TryBreak(Exception ex)
         {
             using (TimedLock.Lock(_lock))
             {
+
                 _lastException = ex;
+                _failCount = Decay(_failCount, _lastFailUpdate);
                 _failCount += 1;
+                _lastFailUpdate = DateTime.UtcNow;

```

Comment:
This library uses `System.Click.UtcNow()` so you should probably use it instead of `DateTime.UtcNow`.

This also would make testing much easier.